test_opensearchpy/test_connection/test_urllib3_http_connection.py

```diff
@@ -0,0 +1,383 @@
+# -*- coding: utf-8 -*-
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+#
+# Modifications Copyright OpenSearch Contributors. See
+# GitHub history for details.
+#
+#  Licensed to Elasticsearch B.V. under one or more contributor
```

Comment:
Do we need this license here?

Reply:
Yes, this code was refactored out of `test_connection.py`, so let's keep it.